### PR TITLE
Remove bootstrap

### PR DESCRIPTION
**Pull request by sh0shinsha**

### CODE REVIEW

File: src/components/group_manager_local.jsx [hunk 1]
```diff
@@ -4,9 +4,11 @@ import Nav from './nav';
 
 import React from 'react';
 import path from 'path';
-import {allGroups, setCurrentGroups, getGroupPhotos, deletePhotoByClassPath, changeGroupName} from '../scripts/media_manager';
+import { allGroups, setCurrentGroups, getGroupPhotos, deletePhotoByClassPath, changeGroupName } from '../scripts/media_manager';
```
Comment (elliotjberman) [inline src/components/group_manager_local.jsx]: nice style

File: src/components/levelguides.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import Round from './packcomponents';
+
+function Level1() {
+    return (
+        <div>
+            <p className="version">Version 1.0.0</p>
+            <Round num="1" mats={[
+                ['BRIK_B', 15]
```
Comment (elliotjberman) [inline src/components/levelguides.jsx]: Just having it on the record that I pushed for an alternate data structure
https://gist.github.com/elliotjberman/4ef92ac86fa244f1554f51c47efd6972

File: src/components/levelguides.jsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                ['FLAK_V', 20],
+                ['LINK_G', 15],
+                ['PIN_BL', 25],
+                ['WOOD', 1]
+            ]} />
+        </div>
+    )
+}
+
+function Level2() {
```
Comment (elliotjberman) [inline src/components/levelguides.jsx]: what's up with these @sh0shinsha

File: src/components/levelguides.jsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    return (
```
Comment (elliotjberman) [inline src/components/levelguides.jsx]: what's up with these @sh0shinsha

File: src/components/main.jsx [hunk 5]
```diff
@@ -11,21 +11,21 @@ export default class Main extends React.Component {
...
     this.levelNumbers = [];
-    for (let i=1; i<=this.length; ++i) {
+    for (let i = 1; i <= this.length; ++i) {
       this.levelNumbers.push(i);
     }
   }
 
   render() {
-    const buttons = this.levelNumbers.map((x) => <Launcher levelNumber={x} key={x} /> );
+    const buttons = this.levelNumbers.map((x) => <Launcher levelNumber={x} key={x} />);
```
Comment (elliotjberman) [inline src/components/main.jsx]: Should actually be `(x => <Launcher ... />)`

File: src/components/settings.jsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import Nav from './nav';
+import LocalStore from '../scripts/local_store';
+
+class Checkbox extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isChecked: LocalStore.getWidescreenSetting(),
```
Comment (elliotjberman) [inline src/components/settings.jsx]: I read this should actually get set in componentDidMount, in which case you would just put it `false` here. Doesn't really matter for now

File: src/img/mats/index.html [hunk 7]
```diff
@@ -0,0 +1,273 @@
+<!DOCTYPE html>
```
Comment (elliotjberman) [inline src/img/mats/index.html]: what's this file on about

File: src/scripts/settings.js [hunk 8]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Checkbox from '../components/settings';
```
Comment (elliotjberman) [inline src/scripts/settings.js]: why not a whole `Settings` component?

File: src/components/levelguides.jsx (hunk 4 above)
Comment (elliotjberman) [reply src/components/levelguides.jsx]: why is Level1 so much bigger than the others

Comment (sh0shinsha) [reply src/components/levelguides.jsx]: those are placeholders for when i actually put in the content

File: src/components/settings.jsx (hunk 6 above)
Comment (sh0shinsha) [reply src/components/settings.jsx]: dont know how to do this one

**Pull request merged**